test/integration/features/basic.feature

```diff
@@ -156,11 +156,11 @@ Feature: Basic test
         And stdout should contain "Removing /etc/NetworkManager/dnsmasq.d/crc.conf file"
         And stdout should contain "Will use root access: removing dnsmasq configuration in /etc/NetworkManager/dnsmasq.d/crc.conf"
         And stdout should contain "Will use root access: executing systemctl daemon-reload command"
-        And stdout should contain "Will use root access: execute systemctl stop/start command"
+        And stdout should contain "Will use root access: execute systemctl reload NetworkManager"
```

Comment:
this should be "Executing systemctl ..." actually in order to match the daemon-reload log, could you change this in `service` in systemd.go? Or I can provide the patch if you prefer.

Reply:
done!